src/components/ui/SuggestsDropdown.jsx

```diff
@@ -83,7 +83,6 @@ const SuggestsDropdown = ({
 
       if (key === 'Enter') {
         if (highlighted !== null) {
-          e.preventDefault();
```

Comment:
I don't think this was useful for something else :D

Reply:
Arf, it was actually necessary to avoid the `keypress` event to be propagated to the input :(  
As a result, a new search will be triggered on Enter, and may select a completely different result from the one that was highlighted.